synapse/storage/databases/main/event_federation.py

```diff
@@ -970,9 +1005,38 @@ def _get_backfill_events(self, txn, room_id, event_list, limit):
 
             event_results.add(event_id)
 
+            txn.execute(
+                connected_insertion_event_query, (event_id, limit - len(event_results))
+            )
+            connected_insertion_event_id_results = list(txn)
```

Comment:
```suggestion
            connected_insertion_event_id_results = txn.fetchall()
```

Does the same thing, but I find it a bit more obvious